matlab_kernel/toolbox/do_matlab_complete.m

```diff
@@ -0,0 +1,37 @@
+function do_complete(substring)
+% Ask for completions of SUBSTRING from MATLAB.
+%
+% This hides the differences between versions
+% for the calls needed to do completions.
+%
+% Copied nearly verbatim from matlab-emacs.
```

Comment:
There was no license in the MATLAB emacs repo when I downloaded it.

Bruno
On Apr 26, 2016 11:15 AM, "Steven Silvester" <notifications@github.com>
wrote:

> In matlab_kernel/toolbox/do_matlab_complete.m
> <https://github.com/Calysto/matlab_kernel/pull/43#discussion_r61136763>:
>
> > @@ -0,0 +1,37 @@
> > +function do_complete(substring)
> > +% Ask for completions of SUBSTRING from MATLAB.
> > +%
> > +% This hides the differences between versions
> > +% for the calls needed to do completions.
> > +%
> > +% Copied nearly verbatim from matlab-emacs.
>
> Can you please include the source license?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Calysto/matlab_kernel/pull/43/files/ebf06f03fb67dc7e59ca9809d6065e8bbcc33a1b#r61136763>
>